packages/create-plugin/src/utils/tests/utils.config.test.ts

```diff
@@ -0,0 +1,119 @@
+import fs from 'fs/promises';
+import path from 'path';
+import { vi } from 'vitest';
+import os from 'os';
+import { getVersion } from '../utils.version.js';
+import { getConfig, UserConfig, CreatePluginConfig } from '../utils.config.js';
+import { DEFAULT_FEATURE_FLAGS } from '../../constants.js';
+
+const mocks = vi.hoisted(() => {
+  return {
+    commandName: 'generate',
+  };
+});
+
+vi.mock('../utils.cli.js', async () => mocks);
+
+const tmpDir = path.join(os.tmpdir(), 'cp-test-config');
```

Comment:
nit: randomize this folder-name or clean it before all tests in case node breaks in the middle of a test you will start the very first test with a dirty temp folder.

Reply:
Good call, changed it in https://github.com/grafana/plugin-tools/pull/754/commits/6fcbe7369f2e062caff1ca93dfd468a201ba2ea1 to clean the folder before each test.